### ISSUE TITLE
Cannot compile snes9x-gtk with USE_SLANG=OFF

### ISSUE THREAD

**Issue by musuruan** [snes9xgit/snes9x#855]
If I try to compile snes9x-gtk v1.62.3 with `USE_SLANG=OFF` I'll get the following error at linking:
```
[...]
/usr/bin/ld: /tmp/cc87jV1W.ltrans0.ltrans.o: in function `S9xOpenGLDisplayDriver::update(unsigned short*, int, int, int)':
/builddir/build/BUILD/snes9x-1.62.3/gtk/src/gtk_display_driver_opengl.cpp:403: undefined reference to `Throttle::set_frame_rate(double)'
/usr/bin/ld: /tmp/cc87jV1W.ltrans0.ltrans.o:/builddir/build/BUILD/snes9x-1.62.3/gtk/src/gtk_display_driver_opengl.cpp:404: undefined reference to `Throttle::wait_for_frame_and_rebase_time()'
collect2: error: ld returned 1 exit status
gmake[2]: Leaving directory '/builddir/build/BUILD/snes9x-1.62.3/gtk/redhat-linux-build'
gmake[2]: *** [CMakeFiles/snes9x-gtk.dir/build.make:1712: snes9x-gtk] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:401: CMakeFiles/snes9x-gtk.dir/all] Error 2
gmake[1]: Leaving directory '/builddir/build/BUILD/snes9x-1.62.3/gtk/redhat-linux-build'
[...]
```

It will compile fine without this option.

**Comment by bearoso** [comment]
I've tried recently to use fewer compile-time options to reduce complexity when adding a lot of runtime options, and in 1.62 and on USE_SLANG isn't supposed to be turned off.

**Comment by lgv5** [comment]
@bearoso sorry to hijack this issue. I'm packaging 1.62.3 in OpenBSD, and for the time being I'm using `USE_SLANG=OFF`. Would you accept a PR to either move `../vulkan/std_chrono_throttle.[ch]pp` outside of `if(USE_SLANG)` (what I'm currently patching for in the package) or to remove `USE_SLANG`, making it mandatory?

Neither is needed for packaging, but I prefer carrying the least local patches.

**Comment by bearoso** [comment]
Fixed in 9be1ee8.
